.gitignore

```diff
@@ -4,3 +4,4 @@ config/static_machine.yml
 /tmp
 /db/*.sqlite3
 /REMEMBER.md
+storage/*
```

Comment:
please keep storage directory in git. Because git cannot keep empty directory create empty file storage/.keep
Put /storage/* in .gitignore